flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -500,10 +500,10 @@
           </tr>
           <tr>
             <td>Interval</td>
-            <td *ngIf="checkPointConfig['interval'] === '0x7fffffffffffffff'">
+            <td *ngIf="checkPointConfig['interval'] == '0x7fffffffffffffff'">
```

Comment:
```suggestion
            <td *ngIf="checkPointConfig['interval'] === disabledInterval">
```

You can utilize the Angular property binding to fix this issue. 

Define `disabledInterval = 0x7fffffffffffffff` in the `JobCheckpointsComponent` class.

Reply:
Thanks for your suggestion! And luckily that we did not make this code merged. @zoltar9264 could you try this proposal and update this PR if valid?

Reply:
Thanks @yangjunhan ! I will hive a try.